dojo/api_v2/serializers.py

```diff
@@ -1055,6 +1055,10 @@ def save(self, push_to_jira=False):
                 sev = item.severity
                 if sev == 'Information' or sev == 'Informational':
                     sev = 'Info'
+                elif sev.lower() == 'moderate':
+                    sev = 'Medium'
+                elif sev.lower() == 'important':
+                    sev = 'High'
```

Comment:
might be best to handle all severities in the same way, i.e. convert to lowercase before comparison?

Reply:
1) I agree with @valentijnscholten comment. If you really want to handle `moderate` and `important` values in API v2 code, you should follow the same logic that handles `Information`/`Informational` values.
2) I think this change is not good. it will impact every parser. We should rely on the parsers to set the severity to the good values of the `dojo/model.py` file:
```Python
    SEVERITIES = {'Info': 4, 'Low': 3, 'Medium': 2,
                  'High': 1, 'Critical': 0}
```
Each tool have his own logic, the contract is that the parser set the value to one of `Info`, `Low`, `Medium`, `High` or `Critical`.

Reply:
Created a [function](https://github.com/DefectDojo/django-DefectDojo/pull/3532/files#diff-3176294e4014378ed7fcff6167cc0a0e5e077c2dd816ebed034e2c5c23e7e754R148) to properly parse the severity. Modified both CSV and JSON parsers to use the function.